packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -222,6 +228,38 @@ export class SelectOutputDir
   }
 }
 
+export class SelectOutputInternalDevDir
+  implements ParametersGatherer<{ outputdir: string }> {
+  public constructor() {}
+
+  public async gather(): Promise<
```

Comment:
This is the code that will scan the project, starting from its root, and display a list of all the folders inside it when the user runs a create command and has entered the metadata's name. 

@ntotten or @nrkruk, I need to know how many folders & levels of depth we need to show when a user is running the create commands so showing a list of available folders is as performant as possible.

Here's a screenshot of the step I'm referring to:
<img width="618" alt="Screen Shot 2019-05-16 at 7 28 18 AM" src="https://user-images.githubusercontent.com/1041842/57861887-478cc980-77ac-11e9-9dbd-e1a9cb9656a7.png">

Reply:
Core modules look like:
core/ui-global-components/components/{namespace}/{name}/{name}.cmp
core/ui-global-components/modules/one/{namespace}/{name}/{name}.html

So we can filter on directories named components for aura commands and directories named modules for lwc commands.

Also, I think we probably just want the right click -> create component on a folder.